libs/cli/licenses/ecs-logging-core-LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Wondering, is there a task to manage these automatically?

Reply:
We don't have anything to automatically grab license/notice files since we don't know where they will come from.